uint8 order_id     
uint8 order_operation   
uint8 table_id                     
run_task/SnackIDWithCount[] snacks # 零食信息
run_task/DrinkIDWithCount[] drinks # 零食信息